pom.xml

```diff
@@ -68,6 +69,12 @@
             <artifactId>display-url-api</artifactId>
             <version>0.2</version>
         </dependency>
+        <dependency> <!-- RequireUpperBoundDeps -->
```

Comment:
Probably should not be here. What are the conflicting dependency trails?

Reply:
See the full commit log: https://github.com/jenkinsci/github-branch-source-plugin/pull/199/commits/fa93cf6794d071451911e3b06577ca82e87a146e